diff --git a/node_modules/wasm-ast-types/main/utils/types.js b/node_modules/wasm-ast-types/main/utils/types.js
index b42659d..869d3a4 100644
--- a/node_modules/wasm-ast-types/main/utils/types.js
+++ b/node_modules/wasm-ast-types/main/utils/types.js
@@ -165,7 +165,7 @@ var getPropertyType = function getPropertyType(context, schema, prop) {
     }
 
     if (_nullableType === 'array' && (0, _typeof2["default"])(info.items) === 'object' && !Array.isArray(info.items)) {
-      type = t.tsArrayType(getType(info.items.type));
+      type = t.tsArrayType(info.items.$ref ? getTypeFromRef(info.items.$ref) : getType(info.items.type));
     } else {
       type = getType(_nullableType);
     }
